src/Context/CachedContextFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Roave\BetterReflection\Context;
```

Comment:
What is the defining criteria for a context? Is it the currently loaded source locators?